utils/setup_nfs_provisioner.sh

```diff
@@ -76,38 +76,44 @@ THEZIP=kubernetes-incubator.zip
 curl -s -L -o $THEZIP https://github.com/kubernetes-incubator/external-storage/archive/master.zip
 if [ $? -ne 0 ]; then
 	echo "ERROR: failed to download external storage zip -- aborting..."
-	exit 6
+	exit 8
 fi
 
+which unzip 2>&1 >/dev/null || yum -y install unzip
```

Comment:
nit `hash` also tests for the existence of an executable and also updates bash's cache of where programs are located.